### PR TITLE
Use build:pack for dbauth when rebuilding the test project

### PR DESCRIPTION
**Pull request by Tobbe**
Now when the dbauth setup is updated (like changes to the template files) they'll be used when rebuilding the test project

### CODE REVIEW

File: tasks/test-project/tui-tasks.js [hunk 1]
```diff
@@ -406,38 +408,76 @@ async function apiTasks(outputPath, { linkWithLatestFwBuild }) {
...
-      .versions.filter((version) => version.includes('canary'))
-      .at(-1)
+    const setupTgz = path.join(setupPkg, 'redwoodjs-auth-dbauth-setup.tgz')
+    const apiTgz = path.join(apiPkg, 'redwoodjs-auth-dbauth-api.tgz')
+    const webTgz = path.join(webPkg, 'redwoodjs-auth-dbauth-web.tgz')
 
-    const dbAuthSetupPath = path.join(
+    const setupTgzDest = path.join(
+      outputPath,
+      'node_modules',
```
Comment (jtoar) [inline tasks/test-project/tui-tasks.js:447]: Ok if we move the tarballs somewhere other than `node_modules`? Like just in the project's root directory? Or even in `.redwood`? Yarn will unpack the tarballs into `node_modules`. I'm not sure if it actually matters or not but I'd rather treat `node_modules` as Yarn's domain

File: tasks/test-project/tui-tasks.js [hunk 2]
```diff
@@ -406,38 +408,76 @@ async function apiTasks(outputPath, { linkWithLatestFwBuild }) {
...
+      'auth-dbauth-setup': './node_modules/auth-dbauth-setup.tgz',
+      'auth-dbauth-api': './node_modules/auth-dbauth-api.tgz',
+      'auth-dbauth-web': './node_modules/auth-dbauth-web.tgz',
+    }
 
     fs.writeFileSync(
-      path.join(dbAuthSetupPath, 'package.json'),
-      JSON.stringify(dbAuthSetupPackageJson, null, 2)
+      projectPackageJsonPath,
+      JSON.stringify(projectPackageJson, null, 2)
```
Comment (jtoar) [inline tasks/test-project/tui-tasks.js:480]: Not a comment on this line in particular, but after this line we run `exec('yarn rw setup auth dbAuth')`. Unless we run `yarn install` first, the tarballs won't be unpacked into `node_modules` and `yarn rw setup` probably won't use the new code

**Review by jtoar — COMMENTED** [review]
One comment on the tarball destination and another on possibly not yarn installing before running `yarn rw setup`

File: tasks/test-project/tui-tasks.js (hunk 2 above)
Comment (Tobbe) [reply tasks/test-project/tui-tasks.js:480]: Good catch. `yarn rw setup` does `yarn install` for the web and api packages. So the new version of those would have been used I think. But not the new version of the setup package itself.

File: tasks/test-project/tui-tasks.js (hunk 1 above)
Comment (Tobbe) [reply tasks/test-project/tui-tasks.js]: Copied to root, and added code to make sure we delete them to keep things clean

**Pull request merged**